README.md

```diff
@@ -6,11 +6,18 @@ You need Ruby 2.3 and Bundler. First, install all the dependencies using `bundle
 ```json
 {}
 ```
-and "rubot-auth" that contains the bot's auth token and client ID on two lines:
+and "rubot-config" that contains the bot's auth token, client ID, GitHub webhook secret, Discord Server ID, Admin role name and the bot name on each line like this:
 ```
 TOKEN HERE
-171456123456123456
+BOT APP ID HERE
+GITHUB WEBHOOK SECRET HERE
```

Comment:
i think that in the current state the bot will fail if the webhook is added to a repo that hasn't got the secret set

you could try checking for for the `X-Hub-Signature` header in the request so that it doesn't happen